### PR TITLE
Improve mobile view: access to the config menu

### LINKED ISSUE #1876: Mobile : No access to parameters
Hi,

on a smartphone, there is no icon to access the settings, would it be possible to fix it?

capture-20180430220226-375x387

### PR DESCRIPTION
**Pull request by math-GH**
Closes #1876

Finally it is possible now to access the configs in mobile view

After:
grafik

grafik

Changes proposed in this pull request:
- shows the normal header (with logo and config button, without the search)
- the menu opens as slider on the right hand side
- the normal view does not change

How to test the feature manually:
1. open FreshRSS in mobile view
2. click on the config button
3. the menu opens

Pull request checklist:
- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

**Comment by math-GH** [comment]
still a draft PR. Please give your first feedback

**Comment by Frenzie** [comment]
Nothing much to say, besides that most of the work is in changing and testing https://github.com/FreshRSS/FreshRSS/issues/1876#issuecomment-449711602

File: p/themes/Ansum/_mobile.scss [hunk 1]
```diff
@@ -69,10 +70,9 @@
 		padding: 0.5rem;
 
 		.item {
-			&.title {
-				display: none;
+			* {
```
Comment (Frenzie) [inline p/themes/Ansum/_mobile.scss]: Is that converted into something more sensible than `*` by the preprocessor? Universal selectors are among the least efficient.

(Then again, it probably doesn't matter much. Still, something to potentially keep an eye on the performance stats on a slow old PC/phone.)

Comment (math-GH) [reply p/themes/Ansum/_mobile.scss]: thanks. Fixed it.

**Comment by math-GH** [comment]
Ready to review

**Comment by Alkarex** [comment]
Very nice improvement @mathContao 👍🏻 
A few bugs:
* I cannot find the search field with most themes: Alternative Dark, Blue Lagoon, Dark, Origin, Flat Design, Pafat...
* The layout with the theme Swage is a bit broken:

Before:

image

After:

image

**Comment by math-GH** [comment]
how can I generate from SCSS file the `RTL` CSS file?

**Comment by Frenzie** [comment]
`make rtl`

(I know I just wrote this [elsewhere](https://github.com/FreshRSS/FreshRSS/discussions/3882#discussioncomment-1540999), but future onlookers might find this.)

**Comment by math-GH** [comment]
>     * I cannot find the search field with most themes: Alternative Dark, Blue Lagoon, Dark, Origin, Flat Design, Pafat...


Only `Ansum` and `Swage` had a search input field in mobile view before.

Swage I had overseen. Now it works fine (except the RTL). Ansum worked fine before.

**Comment by Alkarex** [comment]
> Only `Ansum` and `Swage` had a search input field in mobile view before.

Ah really, I did not realise :-o I think it would be nice to have the search field in all themes, but that can be another PR

**Comment by math-GH** [comment]
anything left to do here?

**Comment by Alkarex** [comment]
Sorry for the delay. Tested again and all is good. Great work 👍🏻

**Pull request merged**